messaging/Controllers/BundlesController.cs

```diff
@@ -104,35 +105,112 @@ public async Task<ActionResult<IncomingMessageItem>> GetIncomingMessageItem(stri
         // POST: Bundles
         // To protect from overposting attacks, see https://go.microsoft.com/fwlink/?linkid=2123754
         [HttpPost]
-        public ActionResult PostIncomingMessageItem(string jurisdictionId, [FromBody] object text, [FromServices] IBackgroundTaskQueue queue)
+        public async Task<ActionResult<Bundle>> PostIncomingMessageItem(string jurisdictionId, [FromBody] object text, [FromServices] IBackgroundTaskQueue queue)

```

Comment:
This change (and associated compiler warning) highlight that `InsertBatchMessages` and `SaveIncomingMessageItem` should be `async` as well, and the save method should call the `SaveChangesAsync` method. But that might require further changes for other relevant code. Do you think that should all be included in this PR?